File: data/imported/knowledge/slides/java/merged/snippets/snippet_065.java
@Test
    void testSaveAndReload() throws IOException {
        try (var pres = new Presentation()) {
            try (var pres2 = saveAndReopen(pres)) {
                assertThat(pres2.getSlides().size()).isEqualTo(1);
            }
        }
    }

File: data/imported/knowledge/slides/net/merged/snippets/snippet_065.cs
[Fact]
    public void TestFirstSlideNumber()
    {
        // first_slide_number persists across save/reload.
        using var pres = new Presentation();
        pres.FirstSlideNumber = 5;
        pres.FirstSlideNumber.Should().Be(5);

        using var pres2 = TestHelpers.SaveAndReopen(pres, _tempDir);
        pres2.FirstSlideNumber.Should().Be(5);
    }